publication-rest/src/main/java/no/unit/nva/publication/update/UpdatePublicationHandler.java

```diff
@@ -86,13 +90,42 @@ public UpdatePublicationHandler(ResourceService resourceService,
         this.uriRetriever = uriRetriever;
     }
 
+    private static boolean isAlreadyPublished(Publication existingPublication) {
+        return PublicationStatus.PUBLISHED.equals(existingPublication.getStatus())
+                || PublicationStatus.PUBLISHED_METADATA.equals(existingPublication.getStatus());
+    }
+
+    private static boolean isPending(TicketEntry publishingRequest) {
+        return TicketStatus.PENDING.equals(publishingRequest.getStatus());
```

Comment:
Er isPending kun gyldig for publishinRequest?

Reply:
Denne er brukt når vi oppretter en ny ticket når vi legger til nye filer. Og ny ticket opprettes kun når det ikke finnes pending publishingRequest fra før av.